.circleci/config.yml

```diff
@@ -47,7 +47,7 @@ jobs:
             sudo pip install -r requirements.txt
       - snyk/scan:
           organization: dwp-dataworks
-          severity-threshold: high
+          severity-threshold: low
```

Comment:
I guess that you've changed this to "low" as nothing appeared in "high", but, for merging in it wants to only be "high" ones that stop the build as agreed.